TrkHitReco/src/StrawHitReco_module.cc

```diff
@@ -52,7 +52,7 @@ namespace mu2e {
       struct Config {
 	fhicl::Atom<int> diag{ Name("diagLevel"), Comment("Diag level"), 0};
 	fhicl::Atom<int> print{ Name("printLevel"), Comment("Print level"), 0};
-	fhicl::Atom<int> fittype { Name( "FitType"), Comment("Waveform Fit Type"), TrkHitReco::FitType::peakminuspedavg};
+	fhicl::Atom<int> fittype { Name( "FitType"), Comment("Waveform Fit Type"), TrkHitReco::FitType::firmwarepmp};
```

Comment:
If a parameter is normally provided by fcl, we do not want default values in the code.   Same comment in many places.

Reply:
I'm ok in removing it. I only changed the value of the default previously set by @bonventre

Reply:
Please also check that the relevant fcl files do set it.

Reply:
Lets defer any changes here.  We can create an issue that the default values should be re-evaluated.

Reply:
Created issue #293